state/protocol/badger/snapshot_test.go

```diff
@@ -222,12 +222,11 @@ func TestSealingSegment(t *testing.T) {
 
 			segment, err := state.AtBlockID(block2.ID()).SealingSegment()
 			require.NoError(t, err)
-
 			// sealing segment should contain B1 and B2
 			// B2 is reference of snapshot, B1 is latest sealed
-			assert.Len(t, segment, 2)
-			assert.Equal(t, block1.ID(), segment[0].ID())
-			assert.Equal(t, block2.ID(), segment[1].ID())
+			assert.Len(t, segment.Blocks, 2)
+			assert.Equal(t, block1.ID(), segment.Blocks[0].ID())
+			assert.Equal(t, block2.ID(), segment.Blocks[1].ID())
```

Comment:
better to add assertion on result list

Reply:
https://github.com/onflow/flow-go/pull/1533/commits/00a77e58055c9b7fbc3a8d569aa91866610f5646